sirepo/package_data/static/js/raydata.js

```diff
@@ -1147,6 +1165,91 @@ SIREPO.app.directive('columnList', function() {
     };
 });
 
+SIREPO.app.directive('scanDetail', function() {
+    return {
+        restrict: 'A',
+        scope: {
+            scan: '<',
+        },
+        template: `
+            <div><strong>Scan Detail</strong></div>
+            <div class="well" style="height: 500px">
+            <div data-ng-if="scan">
```

Comment:
The 500px height makes the layout on a laptop so you can't see the search results easily:

<img width="1470" alt="x" src="https://github.com/radiasoft/sirepo/assets/6516910/4c636528-1d1c-4546-b758-b0dbdbcf6d09">

Ideally, it would be the same height as the search fields to the left, with the option to expand or scroll to see more detail.

Reply:
19b83fcf06f5a7cf22d5b2519a36962ce1026d56